src/main/java/spaceinvaders/Menu.java

```diff
@@ -31,40 +26,50 @@
   private Executor exec;
   private Boolean startedGame = false;
 
+  /**
+   * the constructor for the Menu class.
+   * 
+   * @param ex
+   *          executor for the state pattern
+   */
   public Menu(Executor ex) {
     running = false;
     exec = ex;
 
     JPanel panel = new JPanel();
     panel.setLayout(new GridBagLayout());
-    GridBagConstraints c = new GridBagConstraints();
+    GridBagConstraints gbc = new GridBagConstraints();
```

Comment:
Maybe better to change this to constraints (as I already did for the transition screen)